README.md

```diff
@@ -38,90 +38,132 @@
 </p>
 
 
-# Table of Contents
+**StellarGraph** is a Python library for machine learning on [graphs and networks](https://en.wikipedia.org/wiki/Graph_\(discrete_mathematics\)).
```

Comment:
This is a great concise summary, however the line of text gets a bit lost between the badges and contents.

Reply:
I'll try moving the badges above the `# StellarGraph Machine Learning Library` title, and see if that works better.

Reply:
I think it looks good, thanks!